.babelrc

```diff
@@ -0,0 +1,47 @@
+{
+  "env": {
+    "test": {
+      "presets": [
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions"]
+            }
+          }
+        ],
+        "stage-0"
+      ]
+    },
+    "development": {
+      "presets": [
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 11"]
```

Comment:
Targets the browsers differently in different envs as for testing not as much transpilation might be needed.